sites/kit.svelte.dev/src/routes/__layout.svelte

```diff
@@ -13,12 +16,18 @@
 <SkipLink href="#main" />
 <Nav {page} logo="/images/svelte-kit-horizontal.svg">
 	<svelte:fragment slot="nav-center">
+		{#if browser}
+			<Search />
```

Comment:
The `<li>` is weird, but unfortunately you're right, it's necessary with `<Nav>` as it exists right now. A lot of components in `site-kit` are going to need to be rethought, but doing so would break svelte.dev, so in the short term it's probably easiest to do the weird thing but leave a note